services/src/main/java/org/keycloak/services/resources/admin/permissions/GroupPermissionsV2.java

```diff
@@ -109,78 +100,16 @@ public boolean canManageMembership(GroupModel group) {
             return true;
         }
 
-        return hasPermission(group.getId(), AdminPermissionsSchema.MANAGE_MEMBERSHIP);
+        return eval.hasPermission(new GroupModelRecord(group), null, AdminPermissionsSchema.MANAGE_MEMBERSHIP);
     }
 
     @Override
     public Set<String> getGroupIdsWithViewPermission() {
```

Comment:
We can just make this a no-op method, given that it is not used at all when using FGAP v2?